src/raftstore/store/store.rs

```diff
@@ -155,6 +156,8 @@ pub struct Store<T, C: 'static> {
 
     pub coprocessor_host: Arc<CoprocessorHost>,
 
+    compaction_listener: Option<StdReceiver<CompactedEvent>>,
```

Comment:
Can we send the `CompactionEvent` to the `StoreChannel`?

Reply:
good idea